Dockerfile

```diff
@@ -1,12 +1,28 @@
 FROM golang:1.10
 
-RUN apt-get update && apt-get install -y wget
-RUN wget https://github.com/jwilder/dockerize/releases/download/v0.2.0/dockerize-linux-amd64-v0.2.0.tar.gz
-RUN tar -C /usr/local/bin -xzvf dockerize-linux-amd64-v0.2.0.tar.gz
+ARG PLATFORM=linux
+ENV PLATFORM $PLATFORM_VERSION
+
+ARG ARCH=amd64
+ENV ARCH $ARCH_VERSION
+
+ARG DOCKERIZE_VERSION=v0.2.0
+ENV DOCKERIZE_VERSION $DOCKERIZE_VERSION
+
+ADD https://github.com/jwilder/dockerize/releases/download/$DOCKERIZE_VERSION/dockerize-$PLATFORM-$ARCH-$DOCKERIZE_VERSION.tar.gz /usr/local/bin
+
+RUN cd /usr/local/bin \
+    && tar -xzf ./dockerize-$PLATFORM-$ARCH-$DOCKERIZE_VERSION.tar.gz \
+    && rm -f ./dockerize-$PLATFORM-$ARCH-$DOCKERIZE_VERSION.tar.gz
 
 RUN mkdir -p /go/src/github.com/eventials/goevents
 WORKDIR /go/src/github.com/eventials/goevents
 
-RUN go get github.com/streadway/amqp
-RUN go get github.com/sirupsen/logrus
-RUN go get github.com/stretchr/testify
+RUN go get \
+    github.com/streadway/amqp \
+    github.com/sirupsen/logrus \
+    github.com/stretchr/testify
+
+ENTRYPOINT ["dockerize"]
```

Comment:
i can't remember how this parameter work, but it will permit to do something like `docker-compose run app bash`?

i like entering the container to debug

Reply:
@skrater yes, `dockerize` command is only a wrap to handle some docker details, but you can enter inside container after it runs with `docker-compose exec app bash`, the `exec` command ignore the `ENTRYPOINT`

Reply:
Also, after I forget, if you use `docker-compose run` with the `depends_on` you can run the test's with `docker-compose run app go test -v ./...` because is garanted that the rabbitmq is running with that tag.

Reply:
nice, will merge that, ignore the travis errors (tests and code needs refactor)